src/styles/transformers/rtl.js

```diff
@@ -0,0 +1,90 @@
+import assign from 'lodash.assign';
+import warning from 'warning';
+
+const reTranslate = /((^|\s)translate(3d|X)?\()(\-?[\d]+)/;
+const reSkew = /((^|\s)skew(x|y)?\()\s*(\-?[\d]+)(deg|rad|grad)(,\s*(\-?[\d]+)(deg|rad|grad))?/;
+
+/**
+ * This function ensures that `style` supports both ltr and rtl directions by
+ * checking `styleConstants` in `muiTheme` and replacing attribute keys if
+ * necessary.
+ */
+export default (muiTheme) => (style) => {
+  if (process.env.NODE_ENV !== 'production') {
```

Comment:
hmmm so you mean a transformer that check for every transformer instead of only for rtl... :+1: :+1: Absolutely agree. Love that idea :+1: :+1: